custom/icds_reports/management/commands/rebuild_ccs_record_ucr.py

```diff
@@ -29,10 +50,12 @@ def handle(self):
 
         current_month_start = current_month_start.strftime('%Y-%m-%d')
         last_month_start = last_month_start.strftime('%Y-%m-%d')
-        current_month_doc_ids = CcsRecordMonthly.objects.filter(open_in_month=1,
-                                                                month=current_month_start).values('case_id', 'person_case_id')
-        docs_last_month = CcsRecordMonthly.objects.filter(open_in_month=1,
-                                                          month=last_month_start).values('case_id', 'person_case_id')
+
+        supervisor_ids = AwcLocation.objects.filter(aggregation_level=4).values('supervisor_id')
+        supervisor_ids = list(supervisor_ids)
```

Comment:
should this be a `set` just in case? i suspect there aren't supposed to be any duplicates but maybe wouldn't hurt...

Reply:
supervisors on agg level =4 are unique. Else it will cause error while aggregation.